my_weather.py

```diff
@@ -0,0 +1,39 @@
+'''
+This script prints out the weather for the cities given as arguments.
+Cities should be separated by ','. FE: Dublin,Rome,London
+Optional: add a '-f' flag for Fahrenheit, '-c' flag for Celsius.
+If needed, please change the value of the api key.
+'''
+
+from requests import get
+import json
+import requests
+import sys
+
+# Change api key value to yours if needed
+api_key = 'bc30006e191a5efcc61d86b1db0153bd'
```

Comment:
You should remove this as this is your personal token. on a general note you should always keep tokens and passwords out of source control.